.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I'd prefer not to check this sort of thing into the repo. People can drop their own VSCode settings in there if they want, there's nothing complex about this project.

(Though I wouldn't object to adding this to the `.gitignore` so that adding the file won't leave an untracked entry in the working dir)

As for the build tags: Even if we _do_ add editor configs, I'd argue we shouldn't make the default something that will require the user to install additional software to build and run tests. Both `cleveldb` and `rocksdb` require additional installations locally. I'd suggest folks should have to "opt in" to tags requiring external libraries.